api/src/main/java/org/openmrs/api/context/ServiceContext.java

```diff
@@ -143,10 +143,8 @@ public static ServiceContext getInstance() {
 	 */
 	public static void destroyInstance() {
 		if (ServiceContextHolder.instance != null && ServiceContextHolder.instance.services != null) {
-			if (log.isDebugEnabled()) {
-				for (Map.Entry<Class, Object> entry : ServiceContextHolder.instance.services.entrySet()) {
-					log.debug("Service - " + entry.getKey().getName() + ":" + entry.getValue());
-				}
+			for (Map.Entry<Class, Object> entry : ServiceContextHolder.instance.services.entrySet()) {
```

Comment:
@gracebish we already merged this change and changes to a few other files in this PR. see https://github.com/openmrs/openmrs-core/pull/3400/files . This tells me that you did either not successfully rebase on latest master or you did not push your local changes. Did you get an error message when pushing saying that you would need to force push into this branch?

Reply:
@gracebish you did not address my comment above. There has not been any change since my last review so there is nothing for me to review. Have you tried my suggestions or worked with @HerbertYiga to update this PR on the latest changes on master?

Reply:
@gracebish not sure what happened but now there are ~700 files that have changed and the PR includes a lot of old commits that are already on master. Can you please reach out to someone on talk or @HerberYiga to help you resolve this in a synchronous way like on irc/chat or in a video call with screen sharing. I am on a phone and feel like I cannot help you with the got issues you are currently facing.